### PR TITLE
feat(env-checker): add telemetry event for learn more/continue/cancel buttons

### PR DESCRIPTION
**Pull request by a1exwang**
- Add telemetry events for clicking "Learn More", "Continue" and "Cancel" buttons. One event for each button.
- Also add in function plugin
- Add error event for searchDotnetSdks()

### Test result:
image

https://msazure.visualstudio.com/Microsoft%20Teams%20Extensibility/_workitems/edit/10036370/

### CODE REVIEW

File: packages/vscode-extension/src/debug/depsChecker/vscodeAdapter.ts [hunk 1]
```diff
@@ -51,13 +57,17 @@ export class VSCodeAdapter implements IDepsAdapter {
       continueButton
     );
 
+
```
Comment (meifans) [inline packages/vscode-extension/src/debug/depsChecker/vscodeAdapter.ts:60]: 2 empty line

**Review by meifans — APPROVED** [review]
LGTM

File: packages/vscode-extension/src/debug/depsChecker/vscodeAdapter.ts (hunk 1 above)
Comment (a1exwang) [reply packages/vscode-extension/src/debug/depsChecker/vscodeAdapter.ts]: fixed

**Review by henzhang-ms — APPROVED** [review]
approve for change in function code. looks like only changed object construction.

**Pull request merged**